src/Raven.Studio/wwwroot/App/views/database/tasks/importDatabaseFromFile.html

```diff
@@ -13,7 +13,7 @@ <h3>Import data from a .ravendbdump file into the current database</h3>
             <div class="row">
                 <div class="col-sm-8 col-lg-6 col-xl-5">
                     <div class="input-group file-input" data-bind="validationElement: importedFileName">
-                        <input type="file" id="importDatabaseFilePicker" data-bind="event: { change: _.partial(fileSelected, $element.value) }, disable: isImporting" tabindex="-1" />
+                        <input type="file" accept=".ravendbdump" id="importDatabaseFilePicker" data-bind="event: { change: _.partial(fileSelected, $element.value) }, disable: isImporting" tabindex="-1" />
```

Comment:
Adding this will open the file dialog with 2 file types filters: 

1.  `*.ravendbdump`
2. ` *.*`

It is very convenient. I did the same for the CSV.

![image](https://user-images.githubusercontent.com/17376974/75762034-b7237380-5d42-11ea-9248-0fa9cfa33e67.png)

Reply:
we had this and removed for some reason - please try to find in file history.

Reply:
for csv someone may want to import data from txt file,etc. On linux people tend to forgot about extensions, so please revert this , ok ?

Reply:
leaving this - as discussed